source/addonStore/dataManager.py

```diff
@@ -181,18 +181,20 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 			if NVDAState.shouldWriteToDisk():
 				os.remove(cacheFilePath)
 			return None
+		from json import JSONDecodeError

```

Comment:
please import this from the base file, there is no need for this to be a lazy import

Reply:
Done.